src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Prefer using `const` instead of `let` for `price` since it's not being reassigned but only modified. This helps to avoid unintentional reassignments.